src/web/components/Callout/Checkboxes.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+
+import { CheckboxGroup, Checkbox } from '@guardian/src-checkbox';
+import { FieldLabel } from './FieldLabel';
+
+type Props = {
+    formField: CampaignFieldCheckbox;
+    formData: { [key in string]: any };
+    setFormData: React.Dispatch<React.SetStateAction<{ [x: string]: any }>>;
+};
+
+// We encapsulate the Component's state here instead of directly sourcing it from
+// `formData` to avoid unnessesary rerenders, as this was causing selection animation
+// to be applied whenever formData was changing
+export const Checkboxes = ({ formField, formData, setFormData }: Props) => {
```

Comment:
`Checkboxes` implies multiple checkboxes will be returned but it seems this is a single item

Reply:
agreed, I was not sure what would be a better name, would `CheckboxList` be a better idea? or  `CheckboxGroup`?

Reply:
I thought this component was returning one checkbox, no?

Reply:
no this combines all the checkboxes for that field, it also adds a `FieldLabel` which is on the roadmap to be added to `CheckboxGroup` in `src-*`.

knowing this returns a list of checkboxes, do you think I should still change the name?

Reply:
Oh, these are _options_! This isn't a checkbox at all, it's a radio control

I'd lose the connotation with checkboxes and stick to 'Radio' and 'Option'. To me, a checkbox is the think you check when agreeing to terms and conditions. Not when choosing from a list of options